common/utils/StringUtilsTest.cpp

```diff
@@ -561,6 +561,14 @@ void StringUtilsTest::testStringToUInt() {
   OLA_ASSERT_FALSE(StringToInt("1 bar baz", &value, true));
   OLA_ASSERT_FALSE(StringToInt("65537cat", &value, true));
   OLA_ASSERT_FALSE(StringToInt("4294967295bat bar", &value, true));
+
+  //Base36 test
+  OLA_ASSERT_TRUE(StringToInt("12FZ9A", &value, false, 36));
+  OLA_ASSERT_TRUE(StringToInt("12fz9a", &value2, false, 36));
+  OLA_ASSERT_EQ(value, value2);
+  OLA_ASSERT_TRUE(StringToInt("STRICT1", &value, true, 36));
```

Comment:
This is currently returning false on Travis:
https://travis-ci.org/OpenLightingProject/ola/jobs/606897230#L21660-L21676

Reply:
Indeed, I didn't check numeric values ahead of time STRICT1 overflows an int...

I just ran make on my end I guess that doesn't run the tests because I did not have issues.

Reply:
Yeah you need to do "make check" to actually run the tests.

Reply:
Did that before pushing, none of "my" tests are failing now :)